app/scripts/models/user.js

```diff
@@ -524,6 +530,19 @@ define(function (require, exports, module) {
           }
           return exists;
         });
+    },
+
+    beginFlow: function (flowBeginTime) {
+      if (! this.has('flowId')) {
```

Comment:
The main reason I put it here was so that we don't call `this.set` unless we need to. But there's no harm in setting `flowId` to its current value I guess, it just seems weird is all. I think my preference is for it to be here but if you feel strongly I'll move it over.